pkg/pillar/devicenetwork/addrchange.go

```diff
@@ -167,6 +167,13 @@ func HandleAddressChange(ctx *DeviceNetworkContext) {
 		status, _ := MakeDeviceNetworkStatus(*ctx.DevicePortConfig,
 			dnStatus)
 
+		// Copy port level cloud reachability flag
```

Comment:
Why is this not being done in MakeDeviceNetworkStatus itself? When the status is getting built? Why do we need a separate for loop to do this?